### ISSUE TITLE
Pour la méthode 2- Discuter, agir après, c'est ici;

### ISSUE THREAD

**Issue by lakonis** [catedit-system/catedit-testing#1]
Soumettez des changements ici en expliquant votre point de vue. S'il y a consensus, le groupe décide alors d'effectuer ces changements.

**Comment by Revelli** [comment]
Il semble que les caractéristiques orbitales de tout type constituent un ensemble de sous-catégories qui peuvent influencer l'habitabilité comme par exemple l'inclinaison de l'axe de rotation de la Terre par rapport au plan de l'écliptique permettant d'avoir des saisons ou le fait que la période de rotation sur elle-même de la planète est très petite par rapport à la période de révolution autour de l'étoile

**Comment by lakonis** [comment]
@Revelli, avez vous utilisé l'interface catedit (http://labs.iri-research.org/catedit/) ou l'interface Github ?

**Comment by jagielonka** [comment]
L’habitabilité planétaire peut être en surface et en sous-sol, mais également seulement en sous-sol  pour certaines planètes  ou lunes, peut être faudrait il le préciser sur le graphique   ?

**Comment by vminier** [comment]
Oui, en trouvant une Catégorie et une relation qui précise cela. Par Surface planétaire, Lune, Océan Subglaciaire...

**Comment by vminier** [comment]
@Revelli, totalement d'accord. Il faut donc créer une Catégorie Orbite, puis Inclinaison et mettre en relation Orbite et Inclinaison via "caractérise", et Orbite et Exoplanète via "conditionne"...

**Comment by Revelli** [comment]
Bonsoir,
Excusez mon retard pour répondre mais je rentre de ma réunion de club d'astro et je ne serai pas disponible demain de toute la journée: oui, j'ai suivi la recommandation  (http://labs.iri-research.org/catedit/) qui m'a conduit à m'identifier avec mes données de login GitHub avant de me donner accès aux 2 espaces habilitabilité et testing sans aucun problème pour l'instant

**Comment by 0riona** [comment]
un système solaire doit il être défini par rapport à la catégorie de son étoile type G pour pour le type terre?
et ou 
pour les étoiles naines 
pour définir un "espace limite " d'habitabilité en fonction d'un systeme solaire avec ses planètes

**Comment by vminier** [comment]
Le nom "système solaire" est connecté au soleil. On peut créer un catégorie "systèmes planétaires". Le système solaire serait alors une sous classe de système planétaire.

**Comment by jagielonka** [comment]
En tenant compte de la définition de la vie, il faudrait créer une catégorie de planéte possédant de l'hydrogène moléculaire et du dioxyde de carbone dans son athmosphère ou dans sa constitution propre .L'hydrogénisation du dioxyde de carbone y serait alors possible !

**Comment by vminier** [comment]
On peut en effet créer une planète X en définissant bien ses propriétés. Tester ici. Et passez ensuite sur la zone de travail explorunivers-habitabilite. Revelli a déjà commencé.

**Comment by 0riona** [comment]
donc système planétaire relié à système solaire 
peut on relier à vents "solaire"  et ou champ magnétique planetaire  ( du coup à mouvement des couches interne des planètes ) ?
je me demande à quel moment je peux faire intervenir la gravité ( et ses limites)

**Comment by vminier** [comment]
Oriona, allez donc tester vos idées sur la zone explorunivers-habitabilite. Il faut déclarer la gravité comme une catégorie.

**Comment by worfsonofmogh** [comment]
Bon...disons que je n'avais pas vue "Catedit-testing"

**Comment by worfsonofmogh** [comment]
Excusez-moi...ENCORE, mais préoccupé par la difficulté de connexion ici-bas, inhérente au PC dont je n’ai pas la propriété absolue J’ai oublié l’essentiel : Vous. Cela ne se reproduira plus…
Bref, j'ai créé 3 catégories (pensant qu’ici c’était juste un Test) :
Disque Protoplanétaire + Satellite régulier et irrégulier...Discuter dans "exploreunivers-habitable
Cela étant dit, je pense que parler de tout dans un même post va devenir difficile à cause de la disparité de tous les sujets A abordés et des flots trop long pour se repéré...
A ce titre je me suis encore permis de crée un file de discussion connexe ici : "proposition de Travail"

**Comment by Revelli** [comment]
A Vincent : pourriez-vous SVP répondre à ce fil de discussion sur l'utilisation des documents en ligne? (je n'ai pas encore appréhendé comment saisir des URL dans cet éditeur!)

Excusez-moi de détourner l'objet mais je craignais de ne pas avoir de réponse (et je vois que d'autres commencent à poser le même genre de question - fichier .PDF par exemple)

**Comment by 0riona** [comment]
je ne trouve pas comment éditer une catégorie  et ensuite il faut la décrire puis creer un lien avec la fourchette de causalité et ou effet et ou exemple ?

**Comment by vminier** [comment]
Sur CatEdit, il faut cliquer sur "Créer une catégorie". Ou si la catégorie existe, cliquer sur le crayon. Editer, sauvegarder, soumettre, et re-soumettre en expliquant pourquoi...

**Comment by vminier** [comment]
Pour les définitions, vous pouvez également vous inspirer du glossaire du MOOC.

**Comment by vminier** [comment]
@Revelli, à ma connaissance, on ne peut pas inclure de lien vers un site ou un pdf dans cet éditeur. L'idée est de résumer, d'avoir des définitions suffisantes sans ré-ouvrir vers des sources multiples.

**Comment by worfsonofmogh** [comment]
Je ne sais pas si on a eu une bonne idée de cloner exploreunivers-habitabilité ici.
Alone, avec @Revelli  nous avons pris cette initiative…
cepenant les propriétés ne sont pas identiques
